qiskit/pulse/transforms.py

```diff
@@ -317,6 +317,83 @@ def compress_pulses(schedules: List[Schedule]) -> List[Schedule]:
     return new_schedules
 
 
+def resolve_frames(schedules: List[Schedule], frames: List[List[str]]) -> List[Schedule]:
```

Comment:
I think it's better to take single `schedule`.

Reply:
Changed in a9c2eda